server/app/controllers/checks_controller.py

```diff
@@ -169,6 +182,8 @@ def show_checks(parameters):
                                 'start_date': check.start_date.date()})
         return checks_list
 
+    # This disables the check from triggering MOSS and 
+    # also sets visibility to false in the database.
```

Comment:
If visibility is false, what does that mean?  That MOSS won't be run on it anymore?  That it won't show up in a list?  Please explain.

Reply:
If the visibility is false, the check will not be displayed in the UI.